### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by elmar8287**
- Use store, actions, and reducers in React.
- Connect React and Redux.
- Apply React best practices and language style guides in code.
- Consider [Zeplin`s disign](https://app.zeplin.io/login).

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -1,3 +1,212 @@
...
+  opacity: 0.5;
+  font-size: 0.875rem;
+  color: #121212;
+  padding-left: 2vw;
+}
+
+.under {
+  color: #0290ff;
+  font-weight: 700;
 }
```
Comment (rukundoeric) [inline src/index.css]: - [ ] <b>REQUIRED:</b> There are few things we need to fix to make this app look more like the provided mockup design.

| Original      | Yours |
  | ----------- | ----------- |
| 80047697-0c3f6280-8506-11ea-9416-a4f1800e2329 | scrnli_10_2_2021_10-46-59 PM |
   - The app is not fullscreen. it's like a card in the center of the screen. kindly fix it.
   - Toolbar needs to be given a white background to distinguished it from the rest of the page, just like in the mockup.
   - There must be a profile icon on the right side of the toolbar. Remember, it doesn't need to be functional.
    
Note: Remember, when a client gives a project, you are expected to deliver the product exactly as they requested it. so kindly try to make sure that your app looks like the provided mockup at least 98%.

**Review by rukundoeric — CHANGES_REQUESTED** [review]
Hi @elmar8287, 

_Great job so far 👏 💯 👍. However, there are few issues we need to address before we move on. Please, check the comments below._
## Highlights
  - Follows git workflow.  💯
  - Codebase looks clean and professional. 💯 

## Required Changes ♻️
  - [ ] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
  - [ ] Check the comments under the review.

Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/index.css (hunk 1 above)
Comment (elmar8287) [reply src/index.css]: Fixed

**Review by rukundoeric — CHANGES_REQUESTED** [review]
Hi @elmar8287, 

_Great job so far 👏 💯 👍. You've done well implementing some of the requested changes. However, there are still some which aren't addressed yet. Please, check the comments below._
## Highlights
  - Follows git workflow.  💯
  - Codebase looks clean and professional. 💯 
  - Addressed some of the issues raised from the previous review. 💯 

## Required Changes ♻️
  - [x] You did great, just two more simple things to finalize: 
  
| Original      | Yours |
  | ----------- | ----------- |
| 80047697-0c3f6280-8506-11ea-9416-a4f1800e2329 | scrnli_10_2_2021_11-51-45 PM |
   -  As I mentioned in the previous review, make sure the profile icon is placed on the right side of the toolbar.
   -  Try to change the background of the card to a little bit whiter color so that you can see its edges. or you can use this tool: [ColorPicker](https://chrome.google.com/webstore/detail/colorpick-eyedropper/ohcpnigalekghcmgcdcenkpelffpdolg) to pick the exact color from the mockup design.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
  
  - N/A 

Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by rukundoeric — APPROVED** [review]
Hi @elmar8287,

_Excellent job on this project. Keep it up 🎆_

_Congratulations! 🎉  on the splendid job 👍
You can now merge this PR and submit the project completion form 🚀 . I wish you all the very best in your next tasks 🥇_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
  - [x] some of the colors don't match with the mockup design. checkout this tool: [ColorPicker](https://chrome.google.com/webstore/detail/colorpick-eyedropper/ohcpnigalekghcmgcdcenkpelffpdolg), it helps you pick any color from anywhere on the page. that means if you have a mockup opened in your browser, you can pick up the exact color from it.
  Screen Shot 2021-10-03 at 00 19 04




Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

------
>As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**